pixiebrix/pixiebrix-extension
# #680: Show message on Save if permissions are required in Firefox

Due to the blockers, this is more of an issue to track the exact implementation instead of discussing ad infinitum. 😅 

Blockers:

- [x] https://github.com/pixiebrix/pixiebrix-extension/pull/737#discussion_r666732833
- [x] Decide how/where to show a Grant Permissions button
- [x] https://github.com/pixiebrix/pixiebrix-extension/issues/743

How to test it after the blockers:

1. Open https://app.pixiebrix.com/
2. Create extension with System Notifications brick
3. Click Save
4. See this message:


<img width="890" alt="Screen Shot 6" src="https://user-images.githubusercontent.com/1402241/125073902-d4aadf00-e0e6-11eb-9746-d141ad12979f.png">
